### PR TITLE
cli: fix LinkedPackageResolvePlugin windows tests

### PR DESCRIPTION
**Pull request by freben**
No changeset since it only affects tests.

Supersedes #3851

### CODE REVIEW

File: .github/workflows/master-win.yml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 name: Master Build Windows
 
 on:
+  pull_request:
```
Comment (freben) [inline .github/workflows/master-win.yml:4]: temporary to trigger the master build early, will remove

File: packages/cli/src/lib/bundler/LinkedPackageResolvePlugin.test.ts [hunk 2]
```diff
@@ -50,10 +57,10 @@ describe('LinkedPackageResolvePlugin', () => {
     const callbackX = jest.fn();
     tap(
       {
-        request: '/root/repo/package/x/src/module.ts',
-        path: '/root/repo/package/x/src',
+        request: path.join(root, 'repo', 'package', 'x', 'src', 'module.ts'),
```
Comment (Rugvip) [inline packages/cli/src/lib/bundler/LinkedPackageResolvePlugin.test.ts:60]: Another option here is ´path.resolve(root, 'repo/package/x/src/module.ts')´

Comment (freben) [reply packages/cli/src/lib/bundler/LinkedPackageResolvePlugin.test.ts:60]: Should we use that? Feels odd to use forward slashes but it's for sure more compact

Comment (Rugvip) [reply packages/cli/src/lib/bundler/LinkedPackageResolvePlugin.test.ts:60]: I prefer it :grin:

Resolve essentially does a absolute/relative-path aware join and then normalizes the path, so join is faster but who cares :grin:

File: packages/techdocs-common/src/stages/publish/helpers.test.ts [hunk 3]
```diff
@@ -39,9 +41,11 @@ describe('getHeadersForFileExtension', () => {
 });
 
 describe('getFileTreeRecursively', () => {
+  const root = os.platform() === 'win32' ? 'C:\\rootDir' : '/rootDir';
```
Comment (OrkoHunter) [inline packages/techdocs-common/src/stages/publish/helpers.test.ts:44]: Oh sweet, thank you!

**Review by OrkoHunter — APPROVED** [review]

File: packages/cli/src/lib/bundler/LinkedPackageResolvePlugin.test.ts (hunk 2 above)
Comment (freben) [reply packages/cli/src/lib/bundler/LinkedPackageResolvePlugin.test.ts:60]: Sure! I'll just let the current build finish and then if it's green I'll trigger another round with that change.

**Review by Rugvip — APPROVED** [review]
👌  :shipit:

**Pull request merged**